source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -696,7 +696,7 @@ Pass::Status AggressiveDCEPass::ProcessImpl() {
   // been marked, it is safe to remove dead global values.
   modified |= ProcessGlobalValues();
 
-  // Sanity check.
+  // Coherence check.
```

Comment:
Same, useless comment.

Reply:
Removed - and added a string associated with the assert instead.